test/integration/third-party-wearables-handler/convert-to-model-third-party.ts

```diff
@@ -1,27 +1,26 @@
 import { Entity, Wearable } from '@dcl/schemas'
 import { extractWearableDefinitionFromEntity } from '../../../src/adapters/definitions'
-import { ThirdPartyWearableResponse } from '../../../src/controllers/handlers/third-party-wearables-handler'
-import { ThirdPartyAsset } from '../../../src/types'
 
 type ContentInfo = {
   entities: Entity[]
   contentServerUrl: string
 }
 
 export function convertToThirdPartyWearableResponse(
-  wearables: ThirdPartyAsset[],
+  wearables: any[],
   contentInfo: ContentInfo,
   includeDefinitions: boolean = false
-): ThirdPartyWearableResponse[] {
-  return wearables.map((wearable): ThirdPartyWearableResponse => {
+): any[] {
+  return wearables.map((wearable): any => {
```

Comment:
Is there anything we can use as a `type` here to help us if we need to change something related to this test in the future?